airflow/models.py

```diff
@@ -816,6 +833,40 @@ def __init__(self, dag):
         self.pickle_hash = hash(dag)
         self.pickle = dag
 
+
+class LastDeployedTime(Base, LoggingMixin):
```

Comment:
Like SchedulerState this is a single row, single column entity, which is updated by the deploy automation.